src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String URL = "jdbc:mysql://localhost:3306/library_service";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "root";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+
+    static {
+        try {
+            Class.forName(JDBC_DRIVER);
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't find SQL Driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
```

Comment:
It's a good practice to handle `SQLException` inside the `getConnection` method rather than throwing it, so you can provide more informative error messages and handle specific cases if necessary.